src/phase_net/model.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import nn
+import torchvision
+import pytorch_lightning as pl
+from torchvision import transforms
+
+# Model example for phase recognition training pipeline
+# MIT liscence
+# Author: Yutong Ban, Guy Rosman 
+
+
+
+class CNN_model(pl.LightningModule):
```

Comment:
Does CNN_model have to be a pl.LightningModule? If you're only using the forward operator (true for any pytorch model), are you overconstraining use to use the network only with lightning at deployment time? Alternatively, is a lightning trainer enough?